Resources/Prototypes/_Impstation/Objectives/thief.yml

```diff
@@ -0,0 +1,453 @@
+    stealGroup: ChemDrobe
+  - type: Objective
+    difficulty: 0.7
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: CuraDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Librarian
+  - type: StealCondition
+    stealGroup: CuraDrobe
+  - type: Objective
+    difficulty: 0.4
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: AtmosDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: AtmosphericTechnician
+  - type: StealCondition
+    stealGroup: AtmosDrobe
+  - type: Objective
+    difficulty: 0.7
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: EngiDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: StationEngineer
+  - type: StealCondition
+    stealGroup: EngiDrobe
+  - type: Objective
+    difficulty: 0.7
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: ChefDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Chef
+  - type: StealCondition
+    stealGroup: ChefDrobe
+  - type: Objective
+    difficulty: 0.4
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: DetDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Detective
+  - type: StealCondition
+    stealGroup: DetDrobe
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: JaniDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Janitor
+  - type: StealCondition
+    stealGroup: JaniDrobe
+  - type: Objective
+    difficulty: 0.5
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: SciDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Scientist
+  - type: StealCondition
+    stealGroup: SciDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: RoboDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Scientist
+  - type: StealCondition
+    stealGroup: RoboDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: GeneDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: MedicalDoctor
+  - type: StealCondition
+    stealGroup: GeneDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: ViroDrobeStealObjective
+  components:
+  - type: NotJobRequirement
+    job: MedicalDoctor
+  - type: StealCondition
+    stealGroup: ViroDrobe
+  - type: Objective
+    difficulty: 0.6
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: ServiceTechFabStealObjective
+  components:
+  - type: StealCondition
+    stealGroup: ServiceTechFab
+  - type: Objective
+    difficulty: 1
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: ShiningSpringThiefStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Janitor
+  - type: ObjectiveLimit
+    limit: 3
```

Comment:
not sure why an ObjectiveLimit is imposed only on the shining spring? there can only be up to 3 thieves in a shift anyway, so this is redundant